lib/VideoPlayer/VideoPlayerAccessibilitySupport.js

```diff
@@ -0,0 +1,58 @@
+var
+	kind = require('enyo/kind');
+
+var
+	$L = require('../i18n');
+
+/**
+* @name VideoPlayerAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	initAccessibility: kind.inherit(function (sup) {
+		return function (props) {
+			sup.apply(this, arguments);
+			this.$.jumpBack.set('accessibilityLabel', $L('jumpBack'));
```

Comment:
The convention in Moonstone is to use the English version of the text within `$L()` rather than a key. So 'Fast Forward' instead of 'fastForward'. We should confirm what that text should be as well.